modular_skyrat/modules/sec_haul/code/guns/guns.dm

```diff
@@ -60,7 +56,7 @@
 
 /obj/item/gun/ballistic/automatic/pistol/g18
 	name = "\improper Glock-18"
-	desc = "A burst-firing special anniversary edition of the Glock-18 from Armadyne. Its heavier barrel negatively affects firerate minimally."
+	desc = "A burst-firing special anniversary edition of the Glock-18 from Cantalan Federal Arms. Its heavier barrel negatively affects firerate minimally."
```

Comment:
```suggestion
	desc = "A CFA-made burst firing cheap polymer pistol chambered in 9mm. Its heavy duty barrel affects firerate."
```

Reply:
Also recommend the normal Glock-17 is changed to "polymer pistol" and given the description "A CFA-made cheap polymer pistol chambered in 9mm."